recipe/build.sh

```diff
@@ -26,7 +34,7 @@ else
 	USE_LIBXSMM=ON
 fi
 
-CMAKE_FLAGS="${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release -DPython_EXECUTABLE=${PYTHON}"
+CMAKE_FLAGS="${CMAKE_ARGS} -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release"
```

Comment:
Also CMake says `Python_EXECUTABLE` is unused [on CI]( https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=874072&view=logs&j=bb1c2637-64c6-57bd-9ea6-93823b2df951&t=fb54090b-f7dc-5308-f4ca-2446cf26a524 )

```
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_CUDA_COMPILER
    CUDAToolkit_ROOT
    Python_EXECUTABLE
```

So went ahead and dropped it. Hopefully this cleans up the last of the CMake warnings